verification/src/http_server/handlers/verification/sourcify/api.rs

```diff
@@ -44,6 +44,7 @@ where
     Fut: Future<Output = Result<Response, Error>>,
 {
     for _ in 1..attempts - 1 {
+        // TODO: what if attempts == 0?
```

Comment:
we can accept [std::num::NonZeroUsize](https://doc.rust-lang.org/stable/std/num/struct.NonZeroUsize.html) (or u64)